### ISSUE TITLE
Add an approval step for people creating new groups

### ISSUE THREAD

**Issue by cafelow** [rtcharity/eahub.org#1152]
Hey guys, it would be nice for me to "approve" new groups before they go public, because

1. I've hid inactive groups and groups with no contact info, so people make duplicate groups with "2" at the end of the URL (which is messy).
1. Sometimes people should get some guidance/connections before starting a group. In these cases I might want to hide their newly active group, and it feels a bit nicer to "not approve" these groups than to hide a public group. 

No rush, just a "nice to have" sometime.
